source/documentation/at-rules/import.html.md.erb

```diff
@@ -36,7 +36,7 @@ required to have quotes.
     This makes it very difficult for people (or tools) to tell where anything is
     defined.
 
-  * Because everything's global, libraries must prefix to all their members to
+  * Because everything's global, libraries must add a prefix to all their members to
     avoid naming collisions.
```

Comment:
Please limit each line to 80 characters. Reflow if needed:

```suggestion
  * Because everything's global, libraries must add a prefix to all their
    members to avoid naming collisions.
```